client/ws/ws.go

```diff
@@ -0,0 +1,180 @@
+package ws
+
+import (
+    "fmt"
+    "net/http"
+    "os"
+    "os/exec"
+    "os/signal"
+    sys "golang.org/x/sys/unix"
+    . "github.com/derekparker/delve/client/internal"
+    "github.com/derekparker/delve/command"
+    "github.com/derekparker/delve/goreadline"
+    "github.com/derekparker/delve/proctl"
+    "github.com/gorilla/websocket"
+    "bufio"
+    "log"
+    "sync"
+)
+
+const historyFile string = ".dbg_history"
```

Comment:
I think this got left hanging since it was put into internal, but should go away.